src/lf_apply.rs

```diff
@@ -45,40 +39,56 @@ unsafe fn backup_lpf(
         src_w
     };
     let mut stripe_h = ((64 as c_int) << (cdef_backup & sb128)) - 8 * (row == 0) as c_int >> ss_ver;
-    src = src.offset(((stripe_h - 2) as isize * PXSTRIDE(src_stride)) as isize);
+    src = src.offset((stripe_h - 2) as isize * BD::pxstride(src_stride as usize) as isize);
     if (*(*f).c).n_tc == 1 as c_uint {
         if row != 0 {
             let top = (4 as c_int) << sb128;
             memcpy(
-                &mut *dst.offset(((PXSTRIDE)(dst_stride) * 0 as isize) as isize) as *mut pixel
+                &mut *dst.offset(BD::pxstride(dst_stride as usize * 0) as isize) as *mut BD::Pixel
                     as *mut c_void,
-                &mut *dst.offset(((PXSTRIDE)(dst_stride) * top as isize) as isize) as *mut pixel
-                    as *const c_void,
-                (dst_w << 1) as usize,
+                &mut *dst.offset(BD::pxstride(dst_stride as usize * top as usize) as isize)
+                    as *mut BD::Pixel as *const c_void,
+                if BD::BITDEPTH == 8 {
+                    dst_w as usize
+                } else {
+                    (dst_w << 1) as usize
+                },
             );
             memcpy(
-                &mut *dst.offset(((PXSTRIDE)(dst_stride) * 1 as isize) as isize) as *mut pixel
+                &mut *dst.offset(BD::pxstride(dst_stride as usize * 1) as isize) as *mut BD::Pixel
                     as *mut c_void,
-                &mut *dst.offset(((PXSTRIDE)(dst_stride) * (top + 1) as isize) as isize)
-                    as *mut pixel as *const c_void,
-                (dst_w << 1) as usize,
+                &mut *dst.offset(BD::pxstride(dst_stride as usize * (top + 1) as usize) as isize)
+                    as *mut BD::Pixel as *const c_void,
+                if BD::BITDEPTH == 8 {
+                    dst_w as usize
+                } else {
+                    (dst_w << 1) as usize
+                },
```

Comment:
Anywhere where these are I think are supposed to be `pixel_copy`s or `pixel_set`s.